### ISSUE TITLE
.filtered file is empty even though there are plausible candidates in .all_epitopes file

### ISSUE THREAD

**Issue by alongalor** [griffithlab/pVACtools#451]
pvactools version: 1.3.0
Python version: 3.5.5
Operating System: Linux

**Describe the bug**
`.filtered` file is empty even though there are plausible candidates in `.all_epitopes` file

**To Reproduce**
```
pvacseq run \
-e 8,9,10,11 \
--normal-cov 0 \
--tdna-cov 0 \
--tdna-vaf 0 \
--normal-vaf 0 \
--trna-cov 0 \
--trna-vaf 0 \
input.vcf \
TUMOR \
HLA-A*02:01,HLA-A*26:02,HLA-B*15:01,HLA-B*51:01,HLA-C*01:02,HLA-C*03:03 \
NetMHCpan \
~/test5
```

Attaching `input.vcf` to this issue.

**Log Output**
```
Using TensorFlow backend.
Executing MHC Class I predictions
Converting .vcf to TSV
Completed
Splitting TSV into smaller chunks
Splitting TSV into smaller chunks - Entries 1-1
Completed
Generating Variant Peptide FASTA and Key Files
Generating Variant Peptide FASTA and Key Files - Entries 1-2
Completed
Processing entries for Allele HLA-A*02:01 and Epitope Length 8 - Entries 1-2
Running IEDB on Allele HLA-A*02:01 and Epitope Length 8 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-A*02:01 and Epitope Length 8 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-A*02:01 and Epitope Length 8 - Entries 1-2
Parsing IEDB Output for Allele HLA-A*02:01 and Epitope Length 8 - Entries 1-2 - Completed
Processing entries for Allele HLA-A*02:01 and Epitope Length 9 - Entries 1-2
Running IEDB on Allele HLA-A*02:01 and Epitope Length 9 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-A*02:01 and Epitope Length 9 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-A*02:01 and Epitope Length 9 - Entries 1-2
Parsing IEDB Output for Allele HLA-A*02:01 and Epitope Length 9 - Entries 1-2 - Completed
Processing entries for Allele HLA-A*02:01 and Epitope Length 10 - Entries 1-2
Running IEDB on Allele HLA-A*02:01 and Epitope Length 10 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-A*02:01 and Epitope Length 10 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-A*02:01 and Epitope Length 10 - Entries 1-2
Parsing IEDB Output for Allele HLA-A*02:01 and Epitope Length 10 - Entries 1-2 - Completed
Processing entries for Allele HLA-A*02:01 and Epitope Length 11 - Entries 1-2
Running IEDB on Allele HLA-A*02:01 and Epitope Length 11 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-A*02:01 and Epitope Length 11 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-A*02:01 and Epitope Length 11 - Entries 1-2
Parsing IEDB Output for Allele HLA-A*02:01 and Epitope Length 11 - Entries 1-2 - Completed
Processing entries for Allele HLA-A*26:02 and Epitope Length 8 - Entries 1-2
Running IEDB on Allele HLA-A*26:02 and Epitope Length 8 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-A*26:02 and Epitope Length 8 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-A*26:02 and Epitope Length 8 - Entries 1-2
Parsing IEDB Output for Allele HLA-A*26:02 and Epitope Length 8 - Entries 1-2 - Completed
Processing entries for Allele HLA-A*26:02 and Epitope Length 9 - Entries 1-2
Running IEDB on Allele HLA-A*26:02 and Epitope Length 9 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-A*26:02 and Epitope Length 9 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-A*26:02 and Epitope Length 9 - Entries 1-2
Parsing IEDB Output for Allele HLA-A*26:02 and Epitope Length 9 - Entries 1-2 - Completed
Processing entries for Allele HLA-A*26:02 and Epitope Length 10 - Entries 1-2
Running IEDB on Allele HLA-A*26:02 and Epitope Length 10 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-A*26:02 and Epitope Length 10 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-A*26:02 and Epitope Length 10 - Entries 1-2
Parsing IEDB Output for Allele HLA-A*26:02 and Epitope Length 10 - Entries 1-2 - Completed
Processing entries for Allele HLA-A*26:02 and Epitope Length 11 - Entries 1-2
Running IEDB on Allele HLA-A*26:02 and Epitope Length 11 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-A*26:02 and Epitope Length 11 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-A*26:02 and Epitope Length 11 - Entries 1-2
Parsing IEDB Output for Allele HLA-A*26:02 and Epitope Length 11 - Entries 1-2 - Completed
Processing entries for Allele HLA-B*15:01 and Epitope Length 8 - Entries 1-2
Running IEDB on Allele HLA-B*15:01 and Epitope Length 8 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-B*15:01 and Epitope Length 8 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-B*15:01 and Epitope Length 8 - Entries 1-2
Parsing IEDB Output for Allele HLA-B*15:01 and Epitope Length 8 - Entries 1-2 - Completed
Processing entries for Allele HLA-B*15:01 and Epitope Length 9 - Entries 1-2
Running IEDB on Allele HLA-B*15:01 and Epitope Length 9 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-B*15:01 and Epitope Length 9 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-B*15:01 and Epitope Length 9 - Entries 1-2
Parsing IEDB Output for Allele HLA-B*15:01 and Epitope Length 9 - Entries 1-2 - Completed
Processing entries for Allele HLA-B*15:01 and Epitope Length 10 - Entries 1-2
Running IEDB on Allele HLA-B*15:01 and Epitope Length 10 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-B*15:01 and Epitope Length 10 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-B*15:01 and Epitope Length 10 - Entries 1-2
Parsing IEDB Output for Allele HLA-B*15:01 and Epitope Length 10 - Entries 1-2 - Completed
Processing entries for Allele HLA-B*15:01 and Epitope Length 11 - Entries 1-2
Running IEDB on Allele HLA-B*15:01 and Epitope Length 11 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-B*15:01 and Epitope Length 11 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-B*15:01 and Epitope Length 11 - Entries 1-2
Parsing IEDB Output for Allele HLA-B*15:01 and Epitope Length 11 - Entries 1-2 - Completed
Processing entries for Allele HLA-B*51:01 and Epitope Length 8 - Entries 1-2
Running IEDB on Allele HLA-B*51:01 and Epitope Length 8 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-B*51:01 and Epitope Length 8 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-B*51:01 and Epitope Length 8 - Entries 1-2
Parsing IEDB Output for Allele HLA-B*51:01 and Epitope Length 8 - Entries 1-2 - Completed
Processing entries for Allele HLA-B*51:01 and Epitope Length 9 - Entries 1-2
Running IEDB on Allele HLA-B*51:01 and Epitope Length 9 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-B*51:01 and Epitope Length 9 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-B*51:01 and Epitope Length 9 - Entries 1-2
Parsing IEDB Output for Allele HLA-B*51:01 and Epitope Length 9 - Entries 1-2 - Completed
Processing entries for Allele HLA-B*51:01 and Epitope Length 10 - Entries 1-2
Running IEDB on Allele HLA-B*51:01 and Epitope Length 10 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-B*51:01 and Epitope Length 10 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-B*51:01 and Epitope Length 10 - Entries 1-2
Parsing IEDB Output for Allele HLA-B*51:01 and Epitope Length 10 - Entries 1-2 - Completed
Processing entries for Allele HLA-B*51:01 and Epitope Length 11 - Entries 1-2
Running IEDB on Allele HLA-B*51:01 and Epitope Length 11 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-B*51:01 and Epitope Length 11 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-B*51:01 and Epitope Length 11 - Entries 1-2
Parsing IEDB Output for Allele HLA-B*51:01 and Epitope Length 11 - Entries 1-2 - Completed
Processing entries for Allele HLA-C*01:02 and Epitope Length 8 - Entries 1-2
Running IEDB on Allele HLA-C*01:02 and Epitope Length 8 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-C*01:02 and Epitope Length 8 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-C*01:02 and Epitope Length 8 - Entries 1-2
Parsing IEDB Output for Allele HLA-C*01:02 and Epitope Length 8 - Entries 1-2 - Completed
Processing entries for Allele HLA-C*01:02 and Epitope Length 9 - Entries 1-2
Running IEDB on Allele HLA-C*01:02 and Epitope Length 9 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-C*01:02 and Epitope Length 9 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-C*01:02 and Epitope Length 9 - Entries 1-2
Parsing IEDB Output for Allele HLA-C*01:02 and Epitope Length 9 - Entries 1-2 - Completed
Processing entries for Allele HLA-C*01:02 and Epitope Length 10 - Entries 1-2
Running IEDB on Allele HLA-C*01:02 and Epitope Length 10 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-C*01:02 and Epitope Length 10 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-C*01:02 and Epitope Length 10 - Entries 1-2
Parsing IEDB Output for Allele HLA-C*01:02 and Epitope Length 10 - Entries 1-2 - Completed
Processing entries for Allele HLA-C*01:02 and Epitope Length 11 - Entries 1-2
Running IEDB on Allele HLA-C*01:02 and Epitope Length 11 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-C*01:02 and Epitope Length 11 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-C*01:02 and Epitope Length 11 - Entries 1-2
Parsing IEDB Output for Allele HLA-C*01:02 and Epitope Length 11 - Entries 1-2 - Completed
Processing entries for Allele HLA-C*03:03 and Epitope Length 8 - Entries 1-2
Running IEDB on Allele HLA-C*03:03 and Epitope Length 8 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-C*03:03 and Epitope Length 8 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-C*03:03 and Epitope Length 8 - Entries 1-2
Parsing IEDB Output for Allele HLA-C*03:03 and Epitope Length 8 - Entries 1-2 - Completed
Processing entries for Allele HLA-C*03:03 and Epitope Length 9 - Entries 1-2
Running IEDB on Allele HLA-C*03:03 and Epitope Length 9 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-C*03:03 and Epitope Length 9 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-C*03:03 and Epitope Length 9 - Entries 1-2
Parsing IEDB Output for Allele HLA-C*03:03 and Epitope Length 9 - Entries 1-2 - Completed
Processing entries for Allele HLA-C*03:03 and Epitope Length 10 - Entries 1-2
Running IEDB on Allele HLA-C*03:03 and Epitope Length 10 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-C*03:03 and Epitope Length 10 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-C*03:03 and Epitope Length 10 - Entries 1-2
Parsing IEDB Output for Allele HLA-C*03:03 and Epitope Length 10 - Entries 1-2 - Completed
Processing entries for Allele HLA-C*03:03 and Epitope Length 11 - Entries 1-2
Running IEDB on Allele HLA-C*03:03 and Epitope Length 11 with Method NetMHCpan - Entries 1-2
Running IEDB on Allele HLA-C*03:03 and Epitope Length 11 with Method NetMHCpan - Entries 1-2 - Completed
Parsing IEDB Output for Allele HLA-C*03:03 and Epitope Length 11 - Entries 1-2
Parsing IEDB Output for Allele HLA-C*03:03 and Epitope Length 11 - Entries 1-2 - Completed
Combining Parsed IEDB Output Files
Completed
Running Binding Filters
Completed
Running Coverage Filters
Completed
Running Transcript Support Level Filter
Complete
Running Top Score Filter
Completed
Creating Condensed Report
Completed
Ranking neoepitopes
Completed

Done: Pipeline finished successfully. File TUMOR.filtered.condensed.ranked.tsv contains list of filtered putative neoantigens.

No MHC class II prediction algorithms chosen. Skipping MHC class II predictions.
```

**Output File**

Attaching output below.

**Expected behavior**
Given that the filters are all set to 0 I expect to see some of these make the filtered file. Why are they being filtered out?

Thanks a lot for your help,

Alon
[Archive 3.zip](https://github.com/griffithlab/pVACtools/files/3507219/Archive.3.zip)

**Comment by susannasiebert** [comment]
Version 1.3.0 had a bug in the transcript support level filtering step of the pipeline that was fixed in version 1.3.1 (see the release notes here: https://pvactools.readthedocs.io/en/latest/releases/1_3.html#id2). You might want to consider upgrading to the latest version (1.5.1) to get all the latest features. At a minimum I suggest upgrading to the latest 1.3 hotfix release (1.3.7) so that you get all the bugfixes for the 1.3 feature release.

**Comment by susannasiebert** [comment]
Since there has been no reply to this issue for multiple months, I assume that using the latest version has resolved this problem so I will close this GitHub issue. If you're still running into this issue, please reopen.
